packages/edge-gateway/src/gateway.js

```diff
@@ -110,6 +110,13 @@ export async function gatewayGet (request, env, ctx) {
       const { status, reason } = JSON.parse(denyListResource)
       return new Response(reason || '', { status: status || 410 })
     }
+
+    if (winnerGwResponse.headers.get('content-type') === 'text/html') {
```

Comment:
Let's use `includes` given content type can be values such as `text/html; charset=utf-8`